src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  background: #FFF;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__link {
+  display: inline-block;
+  line-height: 60px;
+  height: 60px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding for your links instead